### PR TITLE
Update to use codegen v0.32.0

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -146,7 +146,7 @@ update/docsy:
 
 .PHONY: gen/code
 gen/code:
-	docker run --rm -v ${PWD}:/repo -it gcr.io/pipecd/codegen:0.8.0 /repo
+	docker run --rm -v ${PWD}:/repo -it ghcr.io/pipe-cd/codegen:v0.32.0 /repo
```
Comment (khanhtc1202) [inline Makefile:149]: should we parse the value from `release/RELEASE` file and use it here?

Comment (khanhtc1202) [reply Makefile:149]: from "hack" script
```shell
╰─ cut -d ' ' -f 2 release/RELEASE
v0.32.0
```

Comment (nghialv) [reply Makefile:149]: What a nice idea! Let me apply it. 💯

**Review by khanhtc1202 — APPROVED** [review]
Nice

**Review by knanao — APPROVED** [review]

**Pull request merged**